Unable to build docker images

Docker Build Command

`docker build --build-arg CPUS=5 `

Error snippet :
`
  CC pipeline/epc_spns_dns.o
/ngic-rtc/dp/kni_pkt_handler.c:20:10: fatal error: **libmnl/libmnl.h: No such file or directory**
 #include <libmnl/libmnl.h>
          ^~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [kni_pkt_handler.o] Error 1
make[2]: *** Waiting for unfinished jobs....
/ngic-rtc/third_party/dpdk/mk/internal/rte.compile-pre.mk:114: recipe for target 'kni_pkt_handler.o' failed
/ngic-rtc/dp/pipeline/epc_arp.c:35:10: fatal error: libmnl/libmnl.h: No such file or directory
 #include <libmnl/libmnl.h>
          ^~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [pipeline/epc_arp.o] Error 1
/ngic-rtc/third_party/dpdk/mk/internal/rte.compile-pre.mk:114: recipe for target 'pipeline/epc_arp.o' failed
/ngic-rtc/third_party/dpdk/mk/rte.extapp.mk:14: recipe for target 'all' failed
make[1]: Leaving directory '/ngic-rtc/dp'
**Makefile:17: recipe for target 'dp' failed**
make[1]: *** [all] Error 2
make: *** [dp] Error 2
The command '/bin/sh -c bash -c "source ./install_builddeps.sh && make -j $CPUS clean && make -j $CPUS EXTRA_CFLAGS='-DUSE_AF_PACKET -ggdb -O2'"' returned a non-zero code: 2
'

--- a/dp/kni_pkt_handler.c
+++ b/dp/kni_pkt_handler.c
@@ -131,7 +131,7 @@ validate_parameters(uint32_t portmask)
 
 #ifdef USE_AF_PACKET
 void
-init_mnl()
+init_mnl(void)
 {
 	int fd;
 	mnl_sock = mnl_socket_open(NETLINK_ROUTE);

--- a/dp/main.h
+++ b/dp/main.h
@@ -294,7 +294,7 @@ extern struct kni_port_params *kni_port_params_array[RTE_MAX_ETHPORTS];
  * Initialize libmnl netlink subsystem
  */
 void
-init_mnl();
+init_mnl(void);
 
 /**
  * Interface to burst rx and enqueue in to kernel

--- a/install_builddeps.sh
+++ b/install_builddeps.sh
@@ -20,6 +20,7 @@ install_pkg_deps() {
 		libnuma-dev \
 		libpcap0.8-dev \
 		libssl-dev \
+                libmnl-dev \
 		libzmq3-dev \
 		wget
 }

--- a/interface/interface.c
+++ b/interface/interface.c
@@ -1213,15 +1213,15 @@ void sig_handler(int signo)
 #ifndef CP_BUILD
 		close(route_sock);
 		cdr_close();
-#endif
+#ifdef USE_AF_PACKET
+		mnl_socket_close(mnl_sock);
+#endif /* USE_AF_PACKET */
+#endif /* CP_BUILD */
 #ifdef TIMER_STATS
 #ifdef AUTO_ANALYSIS
 		print_perf_statistics();
 #endif /* AUTO_ANALYSIS */
 #endif /* TIMER_STATS */
-#ifdef USE_AF_PACKET
-		mnl_socket_close(mnl_sock);
-#endif
 		rte_exit(EXIT_SUCCESS, "received SIGINT\n");
 	}
 	else if (signo == SIGSEGV)

